unfetter-discover/unfetter
# better names for nginx config files

fixes unfetter-discover/unfetter#746

## To Test
* pull this branch in `unfetter` directory
* clean all docker images
* verify all the modes work, by visiting the webapp and looking for errors in the console/webapp
* Test default demo mode
 * `cd unfetter && docker-compose up`
* Test development mode
 * `cd unfetter && docker-compose -f docker-compose.yml -f docker-compose.development up`
* Test deploy mode
 * `cd unfetter && docker-compose -f docker-compose.yml -f docker-compose.deploy up`
